configuration/debian/tedge/postrm

```diff
@@ -1,8 +1,36 @@
 #!/bin/sh
 set -e
 
+
+### Remove user tedge
+if getent passwd tedge >/dev/null; then
+    deluser --quiet --system tedge
+fi
+
 ### Remove configs
+if [ -d "/etc/tedge" ]; then
+    rm -rf /etc/tedge
+fi
```

Comment:
This is actually interesting decision. Since `/etc/tedge` is owned by `tedge` user, we should remove in case `remove` as well as `purge`. However, it contains tedge config file.